xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -516,10 +538,10 @@ void CGUIFontTTF::DrawTextInternal(CGraphicContext& context,
       {
         float nextCursorX = cursorX;
 
-        if (alignment & XBFONT_TRUNCATED)
+        if (alignment & XBFONT_TRUNCATED || alignment & XBFONT_TRUNCATED_LEFT)
```

Comment:
Is this addition necessary? cursorX is initialized with ellipsesWidth in case of left truncation and the iterator of the loop begins at the first character that fits with left truncation

Alternatively mirror the Render loop more closely, which does require this condition and an exception treatment for the first char.

Reply:
removed